src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -380,4 +378,23 @@ TransactionManager::get_extent_if_live_ret TransactionManager::get_extent_if_liv
 
 TransactionManager::~TransactionManager() {}
 
+void TransactionManager::register_metrics()
+{
+  namespace sm = seastar::metrics;
+  metrics.add_group("tm", {
```

Comment:
Just from curiosity: how far are we limited here when it comes to the group naming?

Reply:
just discussed with @rzarzynski offline, we can go with "transaction_manager" for better readability or "tm" for ease of typing and probably just for shorter name. i don't feel strong either way.